samples/traffic_meter/module.yml

```diff
@@ -70,6 +70,49 @@ pipeline:
                   - class_id: ${parameters.detected_object.id}
                     label: ${parameters.detected_object.label}
 
+    # variant group: yolov4 detector
+    # groups are included in the pipeline.elements alongside regular units or other groups
+    - group:
+        # enabled if env var DETECTOR==yolov4
+        init_condition:
+          expr: ${parameters.detector}
+          value: yolov4
+        elements:
+          # below follows detector unit definition as usual
+          - element: nvinfer@detector
+            name: yolov4
+            model:
+              format: onnx
+              remote:
+                url: s3://savant-data/models/yolov4/yolov4.zip
+                checksum_url: s3://savant-data/models/yolov4/yolov4.md5
+                parameters:
+                  endpoint: https://eu-central-1.linodeobjects.com
+              model_file: yolov4.onnx
+              batch_size: ${parameters.batch_size}
+              input:
+                shape: [3, 608, 608]
+                maintain_aspect_ratio: true
+                symmetric_padding: true
+                scale_factor: 0.0039215697906911373
+              output:
+                layer_names: [boxes, confs]
+                converter:
+                  module: savant.converter.yolo_v4
+                  class_name: TensorToBBoxConverter
+                  kwargs:
+                    confidence_threshold: 0.5
+                    top_k: 300
+                objects:
+                  - class_id: ${parameters.detected_object.id}
+                    label: ${parameters.detected_object.label}
+                    selector:
```

Comment:
Please use the same selector and args for each detector. I suggest using MinMaxSizeBBoxSelector instead of the default BBoxSelector, since the converter already performs filtering based on confidence and applies NMS.